src/components/TimerWidgetControls/TimerWidgetControls.module.scss

```diff
@@ -0,0 +1,3 @@
+.buttons Button:nth-child(2) {
```

Comment:
What if you add a third button? You will have to add styles for it too.

Remove this and make the parent flex space-between.